Tools/llm-tool/LLMTool.swift

```diff
@@ -10,16 +10,27 @@ import Tokenizers
 @main
 struct LLMTool: AsyncParsableCommand {
     static var configuration = CommandConfiguration(
-        abstract: "Command line tool for generating text using Llama models",
-        subcommands: [SyncGenerator.self, AsyncGenerator.self],
-        defaultSubcommand: SyncGenerator.self)
+        abstract: "Command line tool for generating text and manipulating LLMs",
+        subcommands: [EvaluateCommand.self],
+        defaultSubcommand: EvaluateCommand.self)
 }
 
-struct LLMArguments: ParsableArguments {
+/// Command line arguments for loading a model.
+struct ModelArguments: ParsableArguments {
```

Comment:
The arguments are now split into logical pieces -- these will be reused in the lora commands.